### DISCUSSION TITLE
Footer Socials

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#160] [General]
# Footer Socials

The footer&rsquo;s socials generates social links.

https://hbstack.dev/en/docs/modules/footer/socials/

**Comment by FuadEfendi** [comment]
strangely, clicking on Email link in Firefox (`gmail = 'mailto:user@example.com'`) opens empty page in Chrome browser on Mac

**Reply by FuadEfendi** [reply]
fixed after configuring default mail client in MacOS
